src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,127 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String VALID_LOGIN = "login";
+    private static final String VALID_PASSWORD = "password";
+    private static final String SHORT_PASSWORD = "pasik";
+    private static final Integer VALID_AGE = 18;
+    private static final Integer INVALID_AGE = 10;
+    private static final String EMPTY_LOGIN = "";
+
+    private static final User userIsNull = null;
```

Comment:
Extracting null to constant is not worth it, you just use it in test. It will be just OK and understandable that in test registerNullUserNotOK you use registerNullUserNotOK(null);